website/frontend/templates/docs/plugin-api.html

```diff
@@ -71,6 +75,36 @@ <h1 id="file_formats">File Formats</h1>
 
       register_format(MyFile)</pre>
 
+    <p>
+    Variables explanation:
+    <ul>
+      <li>"PLUGIN_DESCRIPTION" should be as simple as possible, while still describing the main function.</li>
+      <li>"PLUGIN_VERSION" should be filled with the version of Plugin.</li>
+      <li>"PLUGIN_API_VERSIONS" should be set to the version of Picard this plugin to run with.</li>
+      <li>"PLUGIN_LICENSE" should be set with the license name of the plugin. You can choose on your desired license's abbreviation in <a href="https://spdx.org/licenses/">this license database</a>.</li>
```

Comment:
I'm still not very happy with the wording here, implying that the licenses listed there at any given time are the only ones available. See my previous comment on this for more argumentation: https://github.com/metabrainz/picard-website/pull/136#discussion_r228685639

However, whatever @phw et al are happy with I guess works.